stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py

```diff
@@ -29,4 +29,4 @@ def translate_results(self, data_source, data):
         except Exception as ex:
             raise TranslationResultException("Error when converting results to STIX: {}".format(ex))
         
-        return results
+        return json.dumps(results, indent=4, sort_keys=False)
```

Comment:
we can't change this back to json strings as that might effect how UDI pipeline handles results. you may need to handle this into the proxy module level.

Reply:
okay we'll take a look